apps/studio/components/interfaces/Billing/InvoiceStatusBadge.tsx

```diff
@@ -63,13 +64,21 @@ const InvoiceStatusBadge = ({ status }: InvoiceStatusBadgeProps) => {
               'w-[300px] space-y-2 border border-background',
             ].join(' ')}
           >
-            {[InvoiceStatus.OPEN, InvoiceStatus.UNCOLLECTIBLE].includes(status) && (
-              <p className="text-xs text-foreground">
-                We were not able to collect the money. Make sure you have a valid payment method and
-                enough funds. Outstanding invoices may cause restrictions. You can manually pay the
-                using the "Pay Now" button.
-              </p>
-            )}
+            {[InvoiceStatus.OPEN, InvoiceStatus.ISSUED, InvoiceStatus.UNCOLLECTIBLE].includes(
+              status
+            ) &&
+              (paymentAttempted ? (
+                <p className="text-xs text-foreground">
+                  We were not able to collect the money. Make sure you have a valid payment method
+                  and enough funds. Outstanding invoices may cause restrictions. You can manually
+                  pay the invoice using the "Pay Now" button.
+                </p>
+              ) : (
+                <p className="text-xs text-foreground">
+                  The invoice will soon be charged for. You can manually pay the invoice using the
```

Comment:
For a subscription that uses payment via invoice (no auto collection) but hasn't done the payment yet, but is beyond the due date, this wouldn't be true as we'd never charge for that invoice

Reply:
The text is a bit off even if the payment via invoice is not overdue since we don't actively charge but rely on the customer's action. But back to your actual point. So this kind of invoice (non auto-collection, overdue) is not picked up by our dunning process at some point?

Maybe we should expose the info "auto_collection true/false" via the Mgmt-API rather sooner than later so that we are able to provide spot on texts here. Also to avoid potential support cases.

Maybe for now we can go with something along the lines of

> "The invoice will soon be charged for. In case invoice auto collection is not enabled for your organization, please make sure to make the payment in a timely manner via the link in our invoice emails. You can also pay the invoice now using the "Pay Now" button." ?

Reply:
Subscription endpoint already exposes payment via invoice or card

'card' | 'invoice'

Reply:
But let's not overcomplicate it and just keep the test generic enough to match all cases

Reply:
> The invoice will soon be charged for. In case you pay via invoice instead of card, please make sure to make the payment in a timely manner. You can also pay the invoice using your card now using the "Pay Now" button.